docs/cloud/ops/disk_expand.md

```diff
@@ -0,0 +1,32 @@
+﻿# Expanding disks
+
+Disk can be extended, to accomodate database growth,through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
+
+See also the cloud [sizing guide](/cloud/provision/cloud-instance-guidance/#sizes) for general guidance. 
+
+
+::: note 
+Limitations:  
+* on AWS expanding disks is subject to a rate limit, see [here](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifyVolume.html) for more information.  
+* on Azure, it is currently not possible to expand disks. To increase disk size you need to perform a backup and restore or replicate to a cluster with larger disks.
+:::
+
+
+## Using the Cloud Console
+
+To expand disks in the console, navigate to the clusters view and click on the _Expand Disks_ icon.
```

Comment:
**QUESTION:** Shall we be using capital letters for console or only if we're giving the full name? The same question applies to the command line below.